src/SyncTrayzor/Pages/SettingsView.xaml

```diff
@@ -18,19 +18,20 @@
             <Style TargetType="GroupBox">
                 <Setter Property="Padding" Value="5"/>
                 <Setter Property="Margin" Value="0,5,0,5"/>
-            </Style>    
+            </Style>
         </DockPanel.Resources>
-        
-        <GroupBox DockPanel.Dock="Top" Header="SyncTrayzor">
+
+        <GroupBox DockPanel.Dock="Top" Header="SyncTrayzor" Height="126">
```

Comment:
No worries - I'm happy to fix it up, just didn't want to mess up something you'd done intentionally.